packages/mandelbrot/views/partials/navigation/components.nunj

```diff
@@ -1,5 +1,13 @@
 {% if frctl.components.size %}
-<div class="Navigation-group">
-{{ nav.tree(frctl.components, entity, request) }}
-</div>
+{% set rootCollections = frctl.components.rootCollections().filter('isHidden', false) %}
+{% if rootCollections.size > 0 %}
```

Comment:
This isn’t really useful as we’ll simply loop on nothing on the next line if there’s no collection